rephrase_chempile/rebuild.py

```diff
@@ -0,0 +1,115 @@
+#!/usr/bin/env python3
+"""
+Script to download jablonkagroup/chempile-reasoning dataset,
+add a new 'text' column, and upload back to HuggingFace.
+"""
+
+import os
+from datasets import load_dataset, DatasetDict
+from huggingface_hub import login
+from dotenv import load_dotenv
+
+
+def create_text_column(example):
```

Comment:
But do you really always want to have this `Title/Question/Answer` template? 
Don't we want to have diversity of different assistant-user conversations?

Reply:
Perhaps we also want to have some in different roles to then have templates like https://ollama.com/library/qwen2.5:7b-instruct/blobs/eb4402837c78